pkg/controllers/nodeclass/ami_test.go

```diff
@@ -211,6 +212,21 @@ var _ = Describe("NodeClass AMI Status Controller", func() {
 						},
 					},
 				},
+				{
+					Name: "arm64-nvidia",
+					ID:   "ami-arm64-nvidia",
+					Requirements: []corev1.NodeSelectorRequirement{
+						{
+							Key:      corev1.LabelArchStable,
+							Operator: corev1.NodeSelectorOpIn,
+							Values:   []string{karpv1.ArchitectureArm64},
+						},
+						{
+							Key:      v1.LabelInstanceGPUCount,
```

Comment:
Don't we also need to check the `InstanceAccelerator` label?

Reply:
No, we only add a requirement for the GPU count label, we don't add an inverse requirement for the accelerator label.